libtock/l3gd20.c

```diff
@@ -0,0 +1,140 @@
+#include "l3gd20.h"
+
+typedef struct l3gd20_response {
+  int data1;
+  int data2;
+  int data3;
+  bool done;
+} L3GD20Response;
+
+const float SENZIVITY_FACTOR[3] = {8.75, 17.5, 70.0};
```

Comment:
```suggestion
const float SENSITIVITY FACTOR[3] = {8.75, 17.5, 70.0};
```

Should this be this instead?